lib/rspec/support/spec/stderr_splitter.rb

```diff
@@ -42,8 +43,14 @@ def write(line)
         # Aruba 0.14.2 uses this internally triggering that here
         return if line =~ %r{lib/ruby/2\.7\.0/fileutils\.rb:622: warning:}
 
+        # Ruby 2.7.0 warnings from keyword argments span multiple lines
```

Comment:
Maybe put this closer together with the check for the first line of the warning?
```ruby
return if line =~ %r{^\S+/gems/\S+:\d+: warning:}
```
It took me a second to figure out how are we skipping the first line of it.

Reply:
See change